src/types.ts

```diff
@@ -56,6 +56,7 @@ export interface DeviceContext {
 export interface ParseContext {
   isNegative?: boolean;
   fractions?: boolean;
+  absolute?: boolean
```

Comment:
I'm also not clear what purpose this absolute boolean plays on the ParseContext type... 🤔

Reply:
i might have misunderstood the purpose of ParseContexts, but it seemed to me that `Unit.none` converts the number to rem (`4 => 1rem`). See https://github.com/walidvb/tailwind-react-native-classnames/blob/24522236bf02b36c3b97ff2a7eafb861cd0b7bd2/src/helpers.ts#L221